x/ccv/consumer/module.go

```diff
@@ -173,6 +181,27 @@ func (am AppModule) BeginBlock(ctx sdk.Context, req abci.RequestBeginBlock) {
 // EndBlock implements the AppModule interface
 // Flush PendingChanges to ABCI, send pending packets, write acknowledgements for packets that have finished unbonding.
 func (am AppModule) EndBlock(ctx sdk.Context, req abci.RequestEndBlock) []abci.ValidatorUpdate {
+	if am.keeper.IsPreCCV(ctx) {
```

Comment:
We should have some unit tests or e2e tests validating this behavior

Reply:
Okay

Reply:
Unit tests written